### DISCUSSION TITLE
Live cam instead of black bird in Birdseye?

### DISCUSSION THREAD

**Discussion by kongjudas** [blakeblackshear/frigate#2488] [Ask A Question]
Is there a way of showing a live view as default view instead of the b/w bird picture in Birdseye view? It's a waste of space to just have a huge black spot in the middle of my display unless somethings happening outside. I would like to have my front porch shown all the time everything's quiet. 

Once I select `Frigate JSMpeg` as advised, the basic camera disappears.

**Comment by blakeblackshear** [comment]
You can use continuous mode, but that shows all cameras all the time.

**Comment by kongjudas** [comment]
That would be too small for my wall mount display. I guess I'll have to post a request for the feature, then.
